internal/services/machinelearning/machine_learning_datastore_datalake_gen2_resource_test.go

```diff
@@ -49,6 +50,25 @@ func TestAccMachineLearningDataStoreDataLakeGen2_spn(t *testing.T) {
 	})
 }
 
+func TestAccMachineLearningDataStoreDataLakeGen2_crossSubStorageAccount(t *testing.T) {
+	if os.Getenv("ARM_TEST_ACC_DATASTORE_GEN2_CROSS_SUB_SA_CONTAINER") == "" {
+		t.Skip("ARM_TEST_ACC_DATASTORE_GEN2_CROSS_SUB_SA_CONTAINER not set")
+	}
```

Comment:
Could we remove this and use `ARM_SUBSCRIPTION_ID_ALT` instead for an alternative subscription id?

Reply:
@catriona-m  test case updated.